src/graph/entities/graph_entity.c

```diff
@@ -206,12 +206,17 @@ void GraphEntity_ToString(const GraphEntity *e, char **buffer, size_t *bufferLen
 	*bytesWritten += snprintf(*buffer + *bytesWritten, *bufferLen, "%s", closeSymbole);
 }
 
+inline bool GraphEntity_IsDeleted(const GraphEntity *e) {
+	return DataBlock_ItemIsDeleted(e->entity);
```

Comment:
I feel like accessing the `DataBlock` straight from `graph_entity` breaks abstraction to some extent, seems to me like calling `Graph_EntityIsDeleted` is the way to go, (obviously need to introduce `Graph_EntityIsDeleted`).